content/usage/_index.md

```diff
@@ -11,18 +11,18 @@ Run **@BotKube help** in the Slack channel or directly to the BotKube user to fi
 ![help](/images/help.png)
 As suggested in help message, to execute kubectl commands, send message in following format in the Slack channel where BotKube is already added or as a direct message to BotKube.
 ```
-@BotKube <kubectl command without `kubectl` prefix>
+@BotKube <kubectl command without `kubectl` prefix> [--cluster-name <cluster_name>]
 ```
 See [Examples](/examples/#h-examples) for the use cases.
 
 <h3 class="section-head" id="h-manage-notif"><a href="#h-manage-notif">Managing Slack notifications</a></h3>
 Depending upon your configuration, you will receive Slack notifications about Kubernetes resources lifecycle events and their health.
-BotKube bot allows you to enable/disable notifications using direct messages. Run **@BotKube help** in the Slack channel where the bot is added or as a direct message to the BotKube, the bot will reply with the help message about the supported message formats.
+BotKube bot allows you to enable/disable notifications only in the configured channel. Run **@BotKube help** in the Slack channel where the bot is added or as a direct message to the BotKube, the bot will reply with the help message about the supported message formats.

```

Comment:
notifications only `from`

Reply:
Made requested changes. Updated in the next commit.